test/plugins/CommonIssues.hpp

```diff
@@ -0,0 +1,40 @@
+/**
+ * @file CommonIssues.hpp
+ *
+ * This file contains the definitions of ERS Issues that are common
+ * to two or more of the DAQModules in this package.
+ *
+ * This is part of the DUNE DAQ Software Suite, copyright 2020.
+ * Licensing/copyright details are in the COPYING file that you should have
+ * received with this code.
+ */
+
+#pragma once
+
+#include "appfwk/DAQModule.hpp"
+#include "ers/Issue.hpp"
+
+#include <string>
+
+#define TRACE_NAME "CandidateMaker" // NOLINT
```

Comment:
`TRACE_NAME` specifies the name of the current module for the purposes of debugging output, so it shouldn't go here. I think you can just delete it

Reply:
There is also an `Issues.hpp` header in `include/trigger`. I merged my issues into this file, maybe do the same for the time stamp emulator?